src/services/activityReports.js

```diff
@@ -171,7 +171,7 @@ export async function review(report, status, managerNotes) {
 
 export function activityReportById(activityReportId) {
   return ActivityReport.findOne({
-    attributes: { exclude: ['legacyId'] },
+    attributes: { exclude: ['imported', 'legacyId'] },
```

Comment:
I may need to undo that if we're going to make a frontend view of the un-processed imported data.